permissionless-batches/conf/coordinator/coordinator_run.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/bash
+
+apt update
+apt install -y wget libdigest-sha-perl
+
+# release version
+if [ -z "${SCROLL_ZKVM_VERSION}" ]; then
+  echo "SCROLL_ZKVM_VERSION not set"
+  exit 1
+fi
+
+if [ -z "${HTTP_PORT}" ]; then
+  echo "HTTP_PORT not set"
+  exit 1
+fi
+
+if [ -z "${METRICS_PORT}" ]; then
+  echo "METRICS_PORT not set"
+  exit 1
+fi
+
+case $CHAIN_ID in
+"5343532222") # staging network
+  echo "staging network not supported"
+  exit 1
+  ;;
+"534353") # alpha network
+  echo "alpha network not supported"
+  exit 1
+  ;;
+esac
+
+BASE_DOWNLOAD_DIR="/verifier"
+# Ensure the base directory exists
+mkdir -p "$BASE_DOWNLOAD_DIR"
+
+# Set subdirectories
+ASSETS_DIR="$BASE_DOWNLOAD_DIR/assets"
+OPENVM_DIR="$BASE_DOWNLOAD_DIR/openvm"
+
+# Create necessary directories
+mkdir -p "$ASSETS_DIR"
+mkdir -p "$OPENVM_DIR/verifier"
+
+# Define URLs for asset files
+ASSETS_CHECKSUM_URL="https://circuit-release.s3.us-west-2.amazonaws.com/release-v0.13.1/sha256sum"
```

Comment:
```suggestion
ASSETS_CHECKSUM_URL="https://circuit-release.s3.us-west-2.amazonaws.com/release-v0.13.1/sha256sum"
```

Halo2 related assets are not needed anymore. Thus this script can be dramatically simplified.